Client/src/components/contanier/staff/Student/EditStudent/index.css

```diff
@@ -0,0 +1,33 @@
+.editstudent {
```

Comment:
In all of your work
it should be
`.editStudent`
or
`edit-student`

Reply:
done